spec.bs

```diff
@@ -1045,7 +1139,63 @@ Note: Previously, the `payment` extension allowed for the creation of
               enrollment.https.html
             </wpt>
 
-:  <dfn export>Client extension processing ([=authentication extension|authentication=])</dfn>
+    1. Before step 13 (creating a {{CollectedClientData}}).
+
+        1. Let |bbk_allowed_algorithms| be {{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}.
+
+            1. When
+                {{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}
+                is empty, let |bbk_allowed_algorithms|
+                be
+                {{PublicKeyCredentialCreationOptions}}/{{PublicKeyCredentialCreationOptions/pubKeyCredParams}}.
+
+        1. Let (|bbk| - the platform specific key pair handle, |bbk_id| - an
+            identifier of the |bbk|) be the result of [=creating a key pair=]
+            using |bbk_allowed_algorithms|.
+
+            1. If [=create a key pair=] failed, skip steps below relating to
```

Comment:
Super-nits:

- I think this should still be `creating a key pair` here, grammatically.
- "skip the steps below" (missing the definite article 'the' currently)